stdlib/config/config.lua

```diff
@@ -0,0 +1,152 @@
+--- Config module
+-- @module Config
+
+require 'stdlib/core'
+require 'stdlib/string'
+require 'stdlib/table'
+
+-----------------------------------------------------------------------
+--Setup repeated code for use in sub functions here
+-----------------------------------------------------------------------
+local reservedCharacters = '`~!@#$%^&*+=|;:/\\\'",?()[]{}<>'
+local testReservedCharacters = function(path)
+    local reservedCharacters = reservedCharacters
+    for c in reservedCharacters:gmatch('.') do
+        if path:find(c, 1, true) then
+            return c
+        end
+    end
+    return nil
+end
+
+if not Config then Config = {} end
+Config = {}
+
+--- Creates a new Config object
+-- to ease the management of a config table.
+-- @param config_table [required] The table to be managed.
```

Comment:
Thanks. I very much wanted to add something like this. I had no idea you could do this with ldoc. I, personally, find the ldoc docs to be lacking an a general pain to make heads or tails of.